sdk/schemaregistry/schema-registry-avro/review/schema-registry-avro.api.md

```diff
@@ -8,7 +8,7 @@ import { SchemaRegistry } from '@azure/schema-registry';
 
 // @public
 export class SchemaRegistryAvroSerializer {
-    constructor(registry: SchemaRegistry, schemaGroup: string, options?: SchemaRegistryAvroSerializerOptions);
+    constructor(client: SchemaRegistry, groupName: string, options?: SchemaRegistryAvroSerializerOptions);
     deserialize<T>(buffer: Buffer): Promise<T>;
```

Comment:
This didn't update for some reason like the code change.

Reply:
Done.